src/functions-and-arrays.js

```diff
@@ -29,8 +80,38 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray (wordsUnique) {
```

Comment:
Para este ejercicio la búsqueda de "how to remove duplicates from array js" te devuelve resultados muy interesantes.

https://stackoverflow.com/questions/9229645/remove-duplicate-values-from-js-array
Echale un vistazo. En este punto te recomiendo la solución que proponen con filter. Más adelante te recomiendo la de [...new Set]